oasislmf/model_preparation/reinsurance_layer.py

```diff
@@ -212,8 +204,42 @@ def write_ri_input_files(
         pd.read_csv(ri_info_fp),
         pd.read_csv(ri_scope_fp),
         target_dir,
-        gulsummaryxref=pd.read_csv(gulsummaryxref_fp),
-        fmsummaryxref=pd.read_csv(fmsummaryxref_fp)
+    )
+'''
+
+@oasis_log
+def write_ri_input_files(
+    direct_mapping,
+    exposure_fp,
+    accounts_fp,
+    items_fp,
+    coverages_fp,
+    fm_xref_fp,
+    ri_info_fp,
+    ri_scope_fp,
+    target_dir
+):
+    old_xref_descriptions = pd.DataFrame(generate_xref_descriptions(accounts_fp, exposure_fp))
+    xref_descriptions = direct_mapping
+   
+    ## need to update col names in code --> tmp rename here
+    xref_descriptions['xref_id'] = xref_descriptions['output_id']
+    xref_descriptions['portfolio_number'] = xref_descriptions['portnumber']
```

Comment:
These direct references to OED columns in the code are not good practice, but this seems to be common in the RI code. It's not my implementation, or yours, so I would leave it.

Reply:
same as above, it should get replaced with a call to `merge_oed_to_mapping` and like you noted some external reference to OED columns